apps/docusaurus/docs/integration/wallet-adapter-concept.md

```diff
@@ -52,6 +49,10 @@ This structure offers the following benefits:
 - Simplicity - keeps the Provider package very light and small as the major logic is implemented in the core package.
 - Flexibility - for wallets in creating and implementing custom functions.
 
+:::note
+Following the new [Wallet Standard AIP-62](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-62.md) the wallet adapter supports both standards while the ecosystem is onboarding to the new standard. Dapps can follow the integration guide [here](./wallet-adapter-for-dapp.md#aip-62-wallet-standard) and Wallets can follow the integration guide [here](./wallet-adapter-for-wallets.md#aip-62-wallet-standard)
```

Comment:
Nit: both standards -> the old wallet adapter standard and the new wallet standard described in AIP-62
to make it more explicit